2022-war-in-ukraine.rst

```diff
@@ -0,0 +1,56 @@
+.. post:: March 4, 2022
+   :author: David
+   :location: SAN
+
+
+War in Ukraine and what it means for Read the Docs
+==================================================
+
+With news surrounding the invasion of Ukraine evolving rapidly,
+we felt it was necessary to provide an update to our users and customers.
+
+At Read the Docs, we are outraged and saddened by the invasion of Ukraine
+and we condemn this act of violence as wrong and unlawful.
+We are monitoring the situation in Europe
+and how it relates to our employees, customers, and our services to the open source world.
+
+Read the Docs is a very small company, with only seven employees,
+and fortunately none of them are currently in harm's way.
+Our impact on the open source ecosystem, however, is bigger than our size,
+and we have users all over the world, including in Ukraine and Russia.
+
+
+Impact on our services
+---------
+
+We are continuing to monitor news that could potentially affect our users,
+including sanctions applied to the region.
+
+At this moment, users in Russia and Belarus are likely affected by financial sanctions,
+and subscription payments from these countries may be declined.
+These transactions are impacted by both sanctions targeting banks in the region
+as well as credit card processing limitations.
```

Comment:
I do question whether this paragraph is entirely needed. We do almost no business in those countries. That's why I didn't even mention this previously.

Reply:
This also kind of leaves an unanswered question.. Are we going to cancel those users services for non-payment? Or some other course of action? I'd agree we should remove it if we don't have a policy ready to mention.